### DISCUSSION TITLE
How would I delete local cloud data

### DISCUSSION THREAD

**Discussion by David-Orangemoon** [TurboWarp/packager#199] [Q&A] [answered]
I am trying to find out how to delete local cloud data. I accidentally asked it in the wrong project.

**Comment by GarboMuffin** [comment, marked as answer]
Depends what exactly you're trying to do. If you just want to clear it for yourself for testing, you can open your browser's JavaScript console and run `localStorage.clear()`

There should probably be an easier way to do this
